call-notes/call_5.md

```diff
@@ -0,0 +1,152 @@
+
+**Zantetsu:** Yeah that's kind of what I'm saying like if there's any opportunity for a validators to custom tune their numbers in ways that better match their actual configuration regardless of their stake that is kind of what I was getting at.
+
+**Stephen Akridge:** Yeah, there isn't today in terms of like a command line flag or anything but that potentially could be added if it seemed to be a limiter or helpful.
+
+### SIMD 0047
+
+Video| [13:20](https://youtu.be/AEnkivbha0k?t=800)
+-|-
+
+**Jacob Creech:** In the interest of time, we probably should move on to SIMD that you wanted to talk to. Max, so if you wanted to go ahead and chat about that, we can get it started there.
+
+**Maximilian Schneider:** Yes let me bring it up. can I screen share?
+
+**Jacob Creech:** You should be allowed to let me know if you are not able to.
+
+**Maximilian Schneider:** Okay let's open this file. this is like the newest one we wanted to propose basically. The motivation here is on a high level what do DeFi protocols do when the cluster restarts. I think some protocols have been built in certain slot limits for orders or things like that but especially in I think in lending protocols there is a lot of first come first serve on basically when the network starts up we need to assume that not necessarily all RPC nodes are ready yet like that's something we have seen before is that sometimes a lot of things will not not work on a coordinated restart. So the idea here is well, I think right now it is just a little bit random and uncontrolled how things behave. We can expose to the application developer that there was a controlled restart so this is currently the last time for hard fork on the bank in the reference client. So this is basically just a very simple syscall that you know allows you to access the data and then Implement custom logic. Maybe you want to lock down liquidations for another 100 slots until the Oracles have time to update etcetera. Could be that the chain wasn't a very different state or the price is certain assets were in a very different state before the restart occurred and so this is, it's kind of like just more programmability. The Proposal goes in detail what exactly we want to expose and I think there was some feedback already that this would be also a good mechanism maybe actually expose other things. So just curious before we touch it and create an implementation if there are other data needed than just the slot. It will be good to add it now to the proposal. So we are kind of do one pass over it and it is everything we need.
```

Comment:
```suggestion
**Maximilian Schneider:** Okay let's open this file. This is like the newest one we wanted to propose basically. The motivation here is on a high level what do DeFi protocols do when the cluster restarts. I think some protocols have been built in certain slot limits for orders or things like that but especially in I think in lending protocols there is a lot of first come first serve and basically when the network starts up we need to assume that not necessarily all RPC nodes are ready yet like that's something we have seen before is that sometimes a lot of things will not work on a coordinated restart. So the idea here is well, I think right now it is just a little bit random and uncontrolled how things behave. We can expose to the application developer that there was a controlled restart so this is currently the last hard fork on the bank in the reference client. So this is basically just a very simple syscall that you know allows you to access the data and then implement custom logic. Maybe you want to lock down liquidations for another 100 slots until the Oracles have time to update etcetera. Could be that the chain was at a very different state or the price of certain assets were in a very different state before the restart occurred and so this is, it's kind of like just more programmability. The proposal goes in detail what exactly we want to expose and I think there was some feedback already that this would be also a good mechanism to maybe actually expose other things. So just curious before we touch it and create an implementation if there are other data needed than just the slot. It will be good to add it now to the proposal. So we are kind of do one pass over it and it is everything we need.
```